services/core-api/app/api/now_applications/models/now_application_document_type.py

```diff
@@ -56,11 +56,18 @@ def transform_variables_to_data(now_application, draft_permit, mine, total_liabi
                 now_application.mine_name,
                 'mine_no':
                 now_application.mine_no,
+                'application_type':
+                now_application.notice_of_work_type.description,
                 'proposed_annual_maximum_tonnage':
                 str(now_application.proposed_annual_maximum_tonnage)
                 if now_application.proposed_annual_maximum_tonnage is not None else ' ',
                 'issue_date':
                 format_datetime_to_string(draft_permit.issue_date),
+                'application_dated':
+                format_datetime_to_string(now_application.submitted_date),
+                'application_last_updated_date':
+                format_datetime_to_string(now_application.last_updated_date)
+                if now_application.last_updated_date else now_application.submitted_date,
                 'authorization_end_date':
```

Comment:
I don't see these variables added to the VariableConditionMenu as options, have they been added to the FE too?

Reply:
Removing this, not need to include them.